.all-contributorsrc

```diff
@@ -1,9 +1,19 @@
 {
   "files": [
-    "README.md"
+    "README.basque.md",
```

Comment:
Sorry for not saying this on the first review...
Can we use some sort of glob:RegEx here do we won't need to update this every time, something like `/`/README.*.md`

Reply:
I think I tried that with the `all-contributors` cli, and it didn't seem to support any kind of globbing.
A github action could probably be used in conjunction with the cli to inherit the bash expansion of glob patterns, but that would take a while longer to figure out if it's viable.

Reply:
So that's fine for now!m, Great work